experimental/README.md

```diff
@@ -32,6 +33,13 @@ true
 
 ## Current experimental features
```

Comment:
@BretFisher who can help me build up a list of "past experimental" features? I have no idea where to start.